app/models/user/reputation_tokens/concept_contribution_token.rb

```diff
@@ -0,0 +1,22 @@
+class User::ReputationTokens::ConceptContributionToken < User::ReputationToken
+  params :contributorship, :concept
+  category :authoring
+  reason :contributed_to_concept
+  value 5
```

Comment:
I've made contributing a concept have half the reputation of authoring a concept exercise. Too high still?